metomi/rose/loc_handlers/git.py

```diff
@@ -166,6 +166,9 @@ def _get_commithash(self, remote, ref):
         """
         ret_code, info, _ = self.manager.popen.run(
             "git", "ls-remote", "--exit-code", remote, ref)
+        if ret_code == 128:
+            # repo not found
```

Comment:
Thanks for this 👍, nice change in principle.

I can't find any docs on 128 for ls-remote - the exit code documentation for Git appears to me to be a bit light.

2 is [mentioned](https://www.git-scm.com/docs/git-ls-remote#:~:text=Exit%20with%20status%20%222%22%20when,it%20found%20any%20matching%20refs.) as = "no matching refs are found" . We could consider changing it to `!= 2`?

Reply:
Sounds like a plan.